deploy-edpm.yml

```diff
@@ -47,9 +47,27 @@
   tags:
     - admin-setup
 
-- name: Import run test playbook
+- name: Import run tempest test playbook
```

Comment:
I'm not that comfortable with importing twice the same playbook with different vars. Can't we get the logic directly in the 08-run-test.yml instead?

Reply:
@cjeanner as we checked, this is needed in case a job runs both tempest and tobiko